views/layouts/MainLayout/index.jsx

```diff
@@ -1,14 +1,15 @@
 import React, { useState, useEffect, useContext } from 'react'
 import { Outlet, useNavigate } from 'react-router-dom'
-import { Box } from '@mui/material'
-import Sidebar from '../components/Sidebar/'
+
```

Comment:
```suggestion
```

Reply:
this was intentional to separate node module imports from local imports.